Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1226,11 +1226,22 @@ extension BrowserViewController: URLBarDelegate {
         })
         longPressAlertController.addAction(cancelAction)
 
-        if let popoverPresentationController = longPressAlertController.popoverPresentationController {
-            popoverPresentationController.sourceView = urlBar
-            popoverPresentationController.sourceRect = urlBar.frame
-            popoverPresentationController.permittedArrowDirections = .Any
+        let setupPopover = { [unowned self] in
```

Comment:
The `[unowned self]` here isn't necessary -- the popup is only alive as long as it's in front of the VC. And as long as it's in front of the VC, it will never go out of scope.